_news/2023-02-27-new-survey-on-doa.md

```diff
@@ -0,0 +1,75 @@
+
+While there can be no doubt data quality is important, we believe there is another angle to data-centricity that is often overlooked. And that angle is the software systems that store, process and transform the data. As it turns out, carefully crafted data cleaning process is not worth much if the data that goes into it cannot be easily found and collected. And data management in modern software systems is a big problem, one of the [major obstacles](https://dl.acm.org/doi/full/10.1145/3533378) for rapid adoption of ML in industry. Modern software engineering practices, perfected by generations of engineers building big and small solutions often hinge on data encapsulation - hinding the data behind interfaces, making it secondary to the business logic. And while admittedly these practices advanced the software engineering field quite far, they might not be well equipped to answer the demans of data-driven era. The very idea of hiding data and restricting access to it is orthogonal to the demands of data-driven era around us. Data becomes difficult to find, to collect and to manage almost by design. This turns work of data scientists in industry into nightmare, as any data science related project in a reasonably sized software company starts with [months of searching](https://dl.acm.org/doi/10.1145/2481244.2481247) for bits of data throughout organization.
+
+## Introducing DOA
+
+Data-Oriented Architecture (DOA) is an emerging paradigm that addresses previsely this problem. It is a set of high level principles that serve as general guidance for software engineers that want to build systems where data is highly available, discoverable and verifiable by design, while not sacrifising performance, cost or operational maintenance ovearhead. The [whitepaper](https://community.rti.com/archive/data-oriented-architecture-loosely-coupled-real-time-soa) by Rajive Joshi released in 2007 explains this high level concept excellently, as well as discusses architectural patterns that can be used to achieve DOA in practice. Since this seminal paper was published a few more works came along, discussing general [DOA-like principles](https://ieeexplore.ieee.org/document/8756426) or proposing [concrete implementations](https://dl.acm.org/doi/10.1145/3151759.3151770) in the field. But in general the works that explicitly claim following DOA principles are rare and far in between.
```

Comment:
```suggestion
Data-Oriented Architecture (DOA) is an emerging paradigm that addresses precisely this problem. It is a set of high level principles that serve as general guidance for software engineers that want to build systems where data is highly available, discoverable and verifiable by design, while not sacrifising performance, cost or operational maintenance ovearhead. The [whitepaper](https://community.rti.com/archive/data-oriented-architecture-loosely-coupled-real-time-soa) by Rajive Joshi released in 2007 explains this high level concept excellently, as well as discusses architectural patterns that can be used to achieve DOA in practice. Since this seminal paper was published a few more works came along, discussing general [DOA-like principles](https://ieeexplore.ieee.org/document/8756426) or proposing [concrete implementations](https://dl.acm.org/doi/10.1145/3151759.3151770) in the field. But in general the works that explicitly claim following DOA principles are rare and far in between.
```